indico/core/db/sqlalchemy/principals.py

```diff
@@ -432,3 +433,19 @@ def __eq__(self, other):
         else:
             raise ValueError('Unexpected object type {}: {}'.format(type(other), other))
         return db.and_(self.cls.type == other.principal_type, *criteria)
+
+
+def clone_principals(cls, principals):
+    """Clone a list of principals
```

Comment:
I read recently that the conventions for the docstring require a period at the end of the first line: https://www.python.org/dev/peps/pep-0257/#id17